config/sync/views.view.user_admin_people.yml

```diff
@@ -815,24 +827,23 @@ display:
             default_group: All
             default_group_multiple: {  }
             group_items: {  }
-        uid_raw:
-          id: uid_raw
-          table: users_field_data
-          field: uid_raw
+        roles_target_id_1:
+          id: roles_target_id_1
+          table: user__roles
+          field: roles_target_id
           relationship: none
           group_type: group
           admin_label: ''
           entity_type: user
-          plugin_id: numeric
-          operator: '!='
+          entity_field: roles
+          plugin_id: user_roles
+          operator: not
           value:
-            min: ''
-            max: ''
-            value: '0'
+            patron: patron
           group: 1
           exposed: false
           expose:
-            operator_id: '0'
+            operator_id: ''
```

Comment:
Can you explain these changes to me? Its a bit unclear.

Reply:
I'm filtering all users with the role "patron" out of the view :)

Reply:
Ah, makes sense to not show those. 👍